cmd/livepeer_cli/wizard_transcoder.go

```diff
@@ -41,19 +46,21 @@ func (w *wizard) promptOrchestratorConfig() (float64, float64, int, int, string)
 
 	orch, _, err := w.getOrchestratorInfo()
 	if err != nil || orch == nil {
-		fmt.Println("unable to get current reward cut and fee share")
+		fmt.Println("unable to get current reward cut and fee cut")
 		blockRewardCut = 0
 		feeShare = 0
 	} else {
 		blockRewardCut = eth.ToPerc(orch.RewardCut)
 		feeShare = eth.ToPerc(orch.FeeShare)
 	}
 
-	fmt.Printf("Enter block reward cut percentage (current=%v default=10) - ", blockRewardCut)
+	feeCut := HUNDRED_PERCENT - feeShare
+
+	fmt.Printf("Enter block reward cut percentage (current=%v default=%v) - ", blockRewardCut, defaultRewardCut)
 	blockRewardCut = w.readDefaultFloat(blockRewardCut)
 
-	fmt.Printf("Enter fee share percentage (current=%v default=5) - ", feeShare)
-	feeShare = w.readDefaultFloat(feeShare)
+	fmt.Printf("Enter fee cut percentage (current=%v default=%v) - ", feeCut, defaultFeeCut)
+	feeCut = w.readDefaultFloat(feeCut)
```

Comment:
After this function call, feeCut is never used and `promptOrchestratorConfig` will return feeShare which will always be set to the current on-chain value.